play-ws-standalone/src/main/java/play/libs/ws/StandaloneWSRequest.java

```diff
@@ -32,7 +32,9 @@
      *
      * @return a promise to the response
      */
-    CompletionStage<? extends StandaloneWSResponse> get();
+    default CompletionStage<? extends StandaloneWSResponse> get() {
```

Comment:
It has to be this way because of variance -- if you're extending StandaloneWSRequest with WSRequest (which adds Play-specific types) then you have to specify that the return type is covariant.